charts/README.md

```diff
@@ -0,0 +1,139 @@
+# Karmada
+
+Karmada (Kubernetes Armada) is a Kubernetes management system that enables you to run your cloud-native applications across multiple Kubernetes clusters and clouds, with no changes to your applications. By speaking Kubernetes-native APIs and providing advanced scheduling capabilities, Karmada enables truly open, multi-cloud Kubernetes.
+
+Karmada aims to provide turnkey automation for multi-cluster application management in multi-cloud and hybrid cloud scenarios, with key features such as centralized multi-cloud management, high availability, failure recovery, and traffic scheduling.
+
+## TL;DR
+
+```console
+$ helm install karmada -n karmada-system --create-namespace .
+```
+
+## Prerequisites
+
+- Kubernetes 1.16+
+- helm v3+
+
+## Installing the Chart
+
+To install the chart with the release name `karmada` in namespace `karmada-system`:
+
+```console
+$ helm install karmada -n karmada-system --create-namespace .
+```
+
+If the certificate is `automatically generated`, need to get kubeconfig from the cluster:
```

Comment:
What do you mean `If`? I don't see any description of customizing the certificate.

Reply:
It's my mistake, both modes can be used to obtain kubeconfig in this way, I need to modify the doc. By the way, users can edit values as needs. 
![image](https://user-images.githubusercontent.com/39866845/132276718-00e6345e-1624-4f46-8a63-eef50ea243aa.png)